cypress/support/commands.ts

```diff
@@ -592,13 +594,21 @@ export const createToken = (
   })
 }
 
-export const setupUser = (): Cypress.Chainable<any> => {
+export const setupUser = (useIox: boolean = false): Cypress.Chainable<any> => {
+  useIox = Cypress.env('useIox') || useIox
   const defaultUser = Cypress.env('defaultUser')
-  const userParam = defaultUser ? `?user=${defaultUser}` : ''
+  let params = ''
+  if (useIox) {
+    params = defaultUser
+      ? `?user=${defaultUser}&orgSuffix=ioxlighthouse`
+      : '?orgSuffix=ioxlighthouse'
+  } else {
+    params = defaultUser ? `?user=${defaultUser}` : ''
+  }
```

Comment:
not a huge deal, just a suggestion that using `URLSearchParams` can simplify the logic around needing to figure out if a `?` or `&` is needed for a query param, if you're into that sort of thing:

```ts
const params = new URLSearchParams()
if (defaultUser) {
  params.set('user', defaultUser)
}
if (useIox) {
  params.set('orgSuffix', 'ioxlighthouse')
}

// ...
    url: `/debug/provision?${params.toString()}`
```

Reply:
i like it